homeassistant/components/sensibo/entity.py

```diff
@@ -106,6 +106,8 @@ async def async_send_api_call(
                 self._device_id,
                 params,
             )
+        if command == "reset_filter":
```

Comment:
I'd try to use a decorator for error handling and implement different methods for the different commands instead. This common api call method is not really helping the developer. Typing is lacking.